test/sql/window/test_window_walmart.test_slow

```diff
@@ -0,0 +1,63 @@
+# name: test/sql/window/test_window_walmart.test_slow
+# description: Parallel window lag
+# group: [window]
+
+require vector_size 512
+
+statement ok
+CREATE  TABLE "data" ("Store" INTEGER, "Dept" INTEGER, "Date" DATE, "Weekly_Sales" DOUBLE, "IsHoliday" BOOLEAN);
+
+statement ok
+insert into data select * from read_csv_auto('test/sql/window/walmart.csv.gz');
+
+statement ok
+PRAGMA threads=4
+
+statement ok
```

Comment:
Should the result here be checked?

Reply:
I think the point was it crashed.

Reply:
Fair enough. I think this can be merged then. @hannesmuehleisen any more comments?

Reply:
If it passes the test i had added i'm happy